ARE/services/WebService/src/main/java/eu/asterics/mw/webservice/RestServer.java

```diff
@@ -297,58 +312,81 @@ public String getRuntimeComponentPropertyKeys(@PathParam("componentId") String c
         return response;
     }
 
-    @Path("/runtime/model/components/{componentId}/{componentKey}")
+    @Path("/runtime/model/components/{componentId}/{propertyKey}/dynamicproperty")
```

Comment:
I like `propertyKey` way more than `componentKey`, which is misleading one to think that this property is about a "component-key" (!?) and not about a "property-key".

However, I would not change everything with this PR, because changes like this render the accompanied documentation useless. IMHO it would be better to confine oneself to the problem at hand, and refactor REST paths and the like with a new PR, and update the documentation at the same time.

Reply:
👍  for smaller PRs, however it's not always easy/efficient to split everything up.
just the change of `componentKey` to `propertyKey` shouldn't make existing documentation useless or are you talking about the new `/dynamicproperty` postfix?